x-pack/plugins/metrics_entities/common/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * Base route
+ */
+export const METRICS_ENTITIES_URL = '/api/metrics_entities';
+
+/**
+ * Transforms route
+ */
+export const METRICS_ENTITIES_TRANSFORMS = `${METRICS_ENTITIES_URL}/transforms`;
+
+/**
+ * Global prefix for all the transform jobs
+ */
+export const ELASTIC_NAME = 'estc';
```

Comment:
Shortened to make it easier with the transform 65 character limit? 👍

Reply:
Correct, yes I did.